stories/2021-09-08-Desert-Island-Disks/Desert-Island-Discs.ipynb

```diff
@@ -0,0 +1,2839 @@
+{
```

Comment:
<p>I felt that the network analysis section could perhaps be more concise and it would be nice to have a plot to represent whichever the most interesting aspect of this section is, rather than various printouts of the metrics - to be more informative and in line with other sections</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/alan-turing-institute/TuringDataStories/pull/165/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='94'/>

Reply:
This has been addressed in commit [d5e78cf](https://github.com/alan-turing-institute/TuringDataStories/pull/165/commits/d5e78cf0f0389ae14d548b405b98418fbe013c07)